src/pkg/backup/details/details.go

```diff
@@ -340,29 +374,29 @@ type Details struct {
 }
 
 func (d *Details) add(
-	repoRef, shortRef, parentRef, locationRef string,
+	repoRef path.Path,
+	locationRef *path.Builder,
```

Comment:
we are not using the location to generate shortrefs at the moment and I don't think we can until SDK consumers change their behavior. Otherwise we'll cause breakages by changing what shortref represents when the item's path didn't change

There's also not really much of an interface right now. We can debate about what it should have in the long run, but this PR is already large enough that I don't want to add more to it

Reply:
> we are not using the location to generate shortrefs

Doh, my bad.  I misread the line when I glanced at it.  s/shortReffer/stringer/

> There's also not really much of an interface right now. We can debate about what it should have in the long run, but this PR is already large enough that I don't want to add more to it

For sure, this isn't a current-PR concern.  Per "not much in the interface", that's precisely what I'd hope to make clear.  Give consumers clear understanding that there is not a prefix, item, etc.